src/components/MultiHopTrade/hooks/useActiveQuoteStatus.tsx

```diff
@@ -97,7 +97,7 @@ export const useActiveQuoteStatus = (): QuoteStatus => {
             'trade.errors.noReceiveAddress',
             {
               assetSymbol:
-                lastHopBuyAsset?.symbol ?? translate('trade.errors.buyAssetMiddleSentence'),
+                tradeBuyAsset?.symbol ?? translate('trade.errors.buyAssetMiddleSentence'),
```

Comment:
Nice hop/trade terminology, will use that one in the Osmosis multi-hop implementation as well 🙏🏽 this makes things a lot clearer in a multi-hop context